services/storage-users/pkg/config/config.go

```diff
@@ -173,6 +174,7 @@ type S3NGDriver struct {
 	MaxAcquireLockCycles    int    `yaml:"max_acquire_lock_cycles" env:"STORAGE_USERS_S3NG_MAX_ACQUIRE_LOCK_CYCLES" desc:"When trying to lock files, ocis will try this amount of times to acquire the lock before failing. After each try it will wait for an increasing amount of time. Values of 0 or below will be ignored and the default value of 20 will be used." introductionVersion:"pre5.0"`
 	LockCycleDurationFactor int    `yaml:"lock_cycle_duration_factor" env:"STORAGE_USERS_S3NG_LOCK_CYCLE_DURATION_FACTOR" desc:"When trying to lock files, ocis will multiply the cycle with this factor and use it as a millisecond timeout. Values of 0 or below will be ignored and the default value of 30 will be used." introductionVersion:"pre5.0"`
 	MaxConcurrency          int    `yaml:"max_concurrency" env:"STORAGE_USERS_S3NG_MAX_CONCURRENCY" desc:"Maximum number of concurrent go-routines. Higher values can potentially get work done faster but will also cause more load on the system. Values of 0 or below will be ignored and the default value of 100 will be used." introductionVersion:"pre5.0"`
+	DisableVersioning       bool   `yaml:"disable_versioning" env:"STORAGE_USERS_S3NG_DISABLE_VERSIONING" desc:"Disables versioning for files. When set to true new uploads will overwrite existing files instead of creating a new version." introductionVersion:"6.0"`
```

Comment:
```suggestion
	DisableVersioning       bool   `yaml:"disable_versioning" env:"STORAGE_USERS_S3NG_DISABLE_VERSIONING" desc:"Disables versioning of files. When set to true, new uploads with the same filename will overwrite existing files instead of creating a new version." introductionVersion:"6.0"`
```
Why not adding a global envvar because of `STORAGE_SYSTEM_OCIS_DISABLE_VERSIONING`?